bot/cogs/snekbox.py

```diff
@@ -86,6 +87,11 @@ def rmq(self) -> RMQ:
         if not code:  # None or empty string
             return await ctx.invoke(self.bot.get_command("help"), "eval")
 
+        if all([line.startswith('#') for line in code.strip('\n')]):  # Only comments
```

Comment:
You probably meant `code.split()` here, not `strip`.

The issue though is that this section is before the code has been stripped of the codeblock text, so will always return `False`.

It will also not take into consideration comments that are indented, or more than one comments that have different indent levels.

I kinda don't like the placement of the inline comment `# Only comments` either, as it unnecessarily extends the line length; could do better maybe above the if statement or within it.

Regarding the placement and adjustments required for the comment check, I'll put it in the main review comment.

Reply:
Wait, I just can't read 😅 You are right, I meant to use `split`.

Reply:
~~No, I'm saying you have put `strip` _not_ `split`. You need to use `split`, but your code is using the wrong thing on that line.~~

And you say `I'm aware that content was already stripped` but this part is _not_ where the code is already stripped. That is why I put the recommendations for where things should be placed in the main comment.

Reply:
This is not resolved yet. The only change made at this point is the auto-commit for the recommended code change removing the print `end` kwarg. The rest still needs addressing manually.